test/libsolidity/ASTJSON/assembly/nested_functions.json

```diff
@@ -94,6 +94,7 @@
               }
             ]
           },
+          "functionSelector": "26121ff0",
```

Comment:
Why does this show up now?

Reply:
I did not make full analysis, but daba02b introduces a minor change, setting `AnalysisPerformed` at the end of `CompilerStack::analyze()` always.

And we have code that might be impacted (`m_stackState > CompilerStack::State::ParsedAndImported`)

```
if (_node.isPartOfExternalInterface() && m_stackState > CompilerStack::State::ParsedAndImported)
    attributes.emplace_back("functionSelector", _node.externalIdentifierHex());
```